localize/comments/15/NuGet.Core.resources.dll.lci

```diff
@@ -1,459 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why are you deleting this? 
It's not included in the VSIX, but we will eventually localize the exe, and this DLL is still part of the NuGet.exe